docs/source/services/timelock_service/service-management.rst

```diff
@@ -49,7 +50,7 @@ on TimeLock.
 
 .. code:: bash
 
-      curl -XPOST localhost:8080/timelock/api/new_namespace/timestamp-management/fast-forward?currentTimestamp=TS
+      curl -XPOST https://localhost:8421/timelock/api/new_namespace/timestamp-management/fast-forward?currentTimestamp=TS
```

Comment:
here you refer directly to `https` and `localhost` while in other files you refer to `<protocol>` and `<host>` - probably best to be consistent?

Reply:
As an unrelated comment, it is a bit surprising to me that this method is unauth'd - wouldn't this cause an issue if the endpoint was called with `MAX_LONG`?

Reply:
Yeah, will switch to host/port. 

You're right that there is an issue with long max. However, internally we configure our proxies so that this method is never exposed in a way where it can be called from non-Palantir applications, so that's why this hasn't been prioritised.